Source/Utils/WarpXMovingWindow.cpp

```diff
@@ -81,7 +81,8 @@ WarpX::UpdateInjectionPosition (const amrex::Real a_dt)
             current_injection_position[dir] = pc.m_current_injection_position;
 #endif
 
-            PlasmaInjector* plasma_injector = pc.GetPlasmaInjector();
+            // This only uses the base plasma injector
+            PlasmaInjector* plasma_injector = pc.GetPlasmaInjector(0);
```

Comment:
Basically yes, the moving window would not work in general with multiple sources. Though, it is only using the bulk velocity, so if that was the same for all of the sources, then this would be correct. Here, I punted and just used the first source, which keeps the code consistent with what was already there.
I could add a comment about this. Perhaps a check could be added here to ensure this?